### ISSUE TITLE
Save multiple sonified data from 'soni_obj.play()'

### ISSUE THREAD

**Issue by rdzudzar** [spacetelescope/astronify#45]
Hi, 
I heard the other day about Astronify, what a great project!

As I started exploring and creating sound objects. In Jupyter Notebook I can play simultaneously several files just by having multiple `soni_obj.play()` one after the other, however, I can `.wirte()` only individual files, so I was wondering is it possible to write all into a wile - as it is played?  
I'm using it to sonify galaxy groups - therefore, trying to see how clear would be to hear an entire group of galaxies played/saved at the same time.

**Comment by scfleming** [comment]
Interesting idea!  We'll add this to our list to do some R&D into what's possible.  I'm sure in the mean time you could use audio editing software to overlay multiple sound files on top of each other and then save that as a new single sound file, although I don't know which are free vs. paid, and scriptable vs. not.  If you do any investigation about that let us know if you find a tool that works!

For my own curiousity, from where are your galaxy group data coming from, if you don't mind sharing?

**Comment by rdzudzar** [comment]
Yes, of course, I will look into sound recording/editing, it should be possible, I will keep you updated. 

The data that I'm using is from my observations for PhD research, I have an idea of making a certain properties through Astronify within a Notebook. Once I have a draft I am happy to share with you the Notebook & Data with it.

**Comment by rdzudzar** [comment]
Update:
Created a small project, data and the Jupyter notebook available here: 
https://github.com/rdzudzar/Sonify
Overview here: https://rdzudzar.github.io/sonify.html

**Comment by scfleming** [comment]
N.B. for developers, see also: https://rdzudzar.github.io/sonify.html (similar/same content as above, with embedded solution combining multiple sounds into single file).
